src/aiperf/dataset/memory_map_utils.py

```
@@ -34,39 +38,69 @@
+
+        {base_path}/aiperf_mmap_{benchmark_id}/
+        ├── dataset.dat.zst   # zstd-compressed conversation data
+        └── index.dat.zst     # zstd-compressed index (offsets are for decompressed data)
     """
 
-    def __init__(self, benchmark_id: str | None = None, **kwargs) -> None:
+    def __init__(
+        self,
+        benchmark_id: str | None = None,
+        compress_only: bool = False,
+        **kwargs,
```

Ruff (0.15.2)

[warning] Unused method argument: `kwargs`

(ARG002)